Makefile

```diff
@@ -1,15 +1,23 @@
 #
 # Makefile to run the business analytics software using metabase
 #
+#||: means or true, and prevents the target from crashing if there is an error
```

Comment:
This is a good comment because now I understand what you were doing below.
However what you probably should be doing for this situation is not forcing a 0 return, but instead just [ignoring error returns](https://www.gnu.org/software/make/manual/html_node/Errors.html) from the command which you do in a Makefile with a `-` prefix.